roles/prometheusexporter/templates/deployment.yml.j2

```diff
@@ -83,6 +83,14 @@ spec:
             - mountPath: /config/
               name: config-volume
               readOnly: true
+{% elif type == "probe" %}
+          args:
+            - --config.file=/etc/blackbox_exporter/config.yml
+            - --log.level=debug
```

Comment:
Shouldn't `debug` be a field in the Probe resource?

Reply:
I harcoded it, because for blackbox_exporter, `debug` means showing that probes are being executed:
```
ts=2020-11-16T20:36:36.413Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Beginning probe" probe=http timeout_seconds=5 ts=2020-11-16T20:36:36.413Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Resolving target address" ip_protocol=ip4 
ts=2020-11-16T20:36:36.442Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Resolved target address" ip=147.75.40.148 ts=2020-11-16T20:36:36.442Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Making HTTP request" url=https://147.75.40.148 host=kubernetes.io 
ts=2020-11-16T20:36:36.758Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Received HTTP response" status_code=200 ts=2020-11-16T20:36:36.766Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Response timings for roundtrip" roundtrip=0 start=2020-11-16T20:36:36.442358373Z dnsDone=2020-11-16T20:36:36.442358373Z connectDone=2020-11-16T20:36:36.451026327Z gotConn=2020-11-16T20:36:36.460991651Z responseStart=2020-11-16T20:36:36.758412614Z end=2020-11-16T20:36:36.765988114Z 
ts=2020-11-16T20:36:36.766Z caller=main.go:169 module=http_2xx target=https://kubernetes.io level=debug msg="Probe succeeded" duration_seconds=0.352932335
```
So I can not imagine why would't you be interested in knowing that your configured probes on ServiceMonitors (which make use of your blackbox-exporter, on a separate object), are being executed (for me is like the expected output for `info` loglevel)

Reply:
Seems that's intencional: https://github.com/prometheus/blackbox_exporter/issues/399

Reply:
Ok then, makes sense to have the logs in debug.